addons/crm/views/crm_lead_views.xml

```diff
@@ -642,6 +642,11 @@
                 <xpath expr="//tree" position="attributes">
                     <attribute name="default_order">activity_date_deadline_my</attribute>
                 </xpath>
+                <xpath expr="//tree" position="inside">
+                    <header>
+                        <button name="crm.action_lead_mass_mail" type="action" string="Email" icon="fa-envelope"/>
```

Comment:
FIXME: this is not usually how we do this, but the framework handles it well
The view is defined before the action....